### PR TITLE
2549 add kafka topic schemas to configuration level in the backend

### PR DESCRIPTION
**Pull request by paulseghers**
closes dfds/cloudplatform/issues/2549

# Additional Review Notes
A lot of the work in here is Andreas's from https://github.com/dfds/selfservice-api/pull/156

### CODE REVIEW

File: src/SelfService/Application/ConfigurationLevelService.cs [hunk 1]
```diff
@@ -61,12 +62,27 @@ public void AddMetric(ConfigurationLevelDetail detail)
...
+        _capabilityRepository = capabilityRepository;
+    }
+
     public async Task<ConfigurationLevelInfo> ComputeConfigurationLevel(CapabilityId capabilityId)
     {
         var configLevelInfo = new ConfigurationLevelInfo();
         configLevelInfo.AddMetric(
             new ConfigurationLevelDetail(
-                await GetKafkaTopicConfigurationLevel(),
+                await GetKafkaTopicConfigurationLevel(_kafkaTopicRepository, capabilityId, _messageContractRepository),
```
Comment (andreasfrisch) [inline src/SelfService/Application/ConfigurationLevelService.cs:85]: you don't have to pass the repositories here, I believe

**Review by andreasfrisch — CHANGES_REQUESTED** [review]
minor cleanup

File: src/SelfService/Application/ConfigurationLevelService.cs (hunk 1 above)
Comment (paulseghers) [reply src/SelfService/Application/ConfigurationLevelService.cs:85]: But... I wrote the function to expect those arguments? :thinking: 
You mean use them as attributes of the class instead?

Comment (paulseghers) [reply src/SelfService/Application/ConfigurationLevelService.cs]: ok I see. Done

**Review by andreasfrisch — APPROVED** [review]

**Pull request merged**